sarif-2.2/prose/edit/src/file-format-58-notification-object.md

```diff
@@ -98,3 +100,12 @@ A `notification` object **MAY** contain a property named `timeUtc` whose value i
 If the notification is a result of a runtime exception, the `notification` object **MAY** contain a property named `exception` whose value is an `exception` object ([sec](#exception-object)).
 
 If the notification is not the result of a runtime exception, the `exception` property **SHALL** be absent.
+
+### relatedLocations property{#notification-object--relatedlocations-property}
+
+A `notification` object **SHOULD** contain a property named `relatedLocations` whose value is an array of zero or more unique ([sec](#array-properties-with-unique-values)) `location` objects ([sec](#location-object)) that identify those locations to which the condition described by the notification applies.
```

Comment:
Interesting point. I will add the property to the object "of this task" as non-required but I propose to the TC to discuss the consistency of requesting the same property in two different places and tow distinct compliance levels.

> Personally I expect JSON instances to omit keys in case the values are empty containers or otherwise would be `null` if the schema allows. A schema should serve the communication of instances and not the other way around. Providing noise when there is no information to convey should be required on grounds of real use cases where that actually help(ed). 

@dmk42 Propose to be discussed in the TC meeting 2024-07-11